backend/distributor/api.py

```diff
@@ -168,13 +169,26 @@ class ContactMessageAPIView(APIView):
     @staticmethod
     def post(request, *args, **kwargs):
         serializer = ContactMessageSerializer(data=request.data)
-        if serializer.is_valid():
-            ContactMessage.objects.create(
-                full_name=serializer.data['full_name'],
-                phone_number=serializer.data['phone_number'],
-                email=serializer.data['email'],
-                title=serializer.data['title'],
-                body=serializer.data['body'],
-            )
-            return Response({'success': True}, status=status.HTTP_201_CREATED)
-        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+        serializer.is_valid(raise_exception=True)
```

Comment:
Why do we raise exception, instead of returning normal response?

Reply:
It already returns normal response, I just reduce code lines, nothing has changed, it remains as before. Instead of checking inside if statement we can set `raise_exception=True` and it will return `Bad Request` with `status code 400` and response body will be with detailed error messages